vendor/atlantis/Makefile

```diff
@@ -7,7 +7,10 @@ export DOWNLOAD_URL ?= $(PACKAGE_REPO_URL)/releases/download/v$(PACKAGE_VERSION)
 export APK_BUILD_TEMPLATE ?= APKBUILD.github-binary
 
 install:
```

Comment:
Could you move this to a macro?

Reply:
https://github.com/cloudposse/packages/blob/master/tasks/Makefile.package#L31-L56

Reply:
Done 👍